src/plugins/data_view_editor/public/data_view_editor_service.ts

```diff
@@ -308,8 +308,12 @@ export class DataViewEditorService {
     getFieldsOptions: GetFieldsOptions,
     requireTimestampField: boolean
   ) => {
-    const fields = await ensureMinimumTime(this.dataViews.getFieldsForWildcard(getFieldsOptions));
-    return extractTimeFields(fields as DataViewField[], requireTimestampField);
+    try {
+      const fields = await ensureMinimumTime(this.dataViews.getFieldsForWildcard(getFieldsOptions));
+      return extractTimeFields(fields as DataViewField[], requireTimestampField);
+    } catch (e) {
+      return [];
+    }
```

Comment:
@mattkime When debugging those tests, I noted an uncaught exception, resolved by this code, any objections adding this?